### ISSUE TITLE
6.0 Feedback:  <Robot> is busy modal appears twice after clicking on clear and proceed to setup button

### ISSUE THREAD

**Issue by reachojhaneha** [Opentrons/opentrons#10099]
### Overview

If the user chooses a robot that has a protocol loaded and in the not started, complete, canceled or failed state,  **robot is busy**  modal appears

### Current Behavior

this modal appears twice, second time it vanishes in seconds
![Screen Shot 2022-04-26 at 5 34 50 PM](https://user-images.githubusercontent.com/39049853/165396339-17ff2e29-6cb6-4253-881d-741ef6c7e2af.png)

after clicking on clear and proceed to setup, again a modal appears with robot is busy and disappears
![Screen Shot 2022-04-26 at 5 35 35 PM](https://user-images.githubusercontent.com/39049853/165396429-b85f380f-da6b-4f4a-8cf0-861144bed026.png)

### Expected Behavior

This modal should appear only once , and after clicking on **clear and proceed to setup** button -> user to be able to see the new protocol on the selected robot

### Steps To Reproduce

1. Select a protocol on protocol page
2. Click on the overflow button -> click on run -> choose a robot that already has a protocol loaded -> click on proceed to setup
3. "Robot is busy" modal appears
4. User still wants to use the robot and clear the previous loaded protocol -> clicking on **clear and proceed to setup** button should take me to setup tab
5. But now the modal reappears and vanishes again

https://user-images.githubusercontent.com/39049853/165397100-d912e42f-b741-492b-abd8-ed6f758af40c.mov

Note: at 13sec pop-up appears twice

**Comment by nusrat813** [comment]
👀  from dev needed for next steps

**Comment by sakibh** [comment]
Closing this ticket because it was addressed in a separate issue where the robot is busy modal was removed and the protocol will automatically clear and proceed to setup.
